### PR TITLE
add exercise 14 files

### PR DESCRIPTION
**Pull request by pepito-alonajane**
Please review my file. Thank you!

### CODE REVIEW

File: pepito_alonajane/countries.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="countries.css">
+    <title>Countries</title>
+</head>
+<body>
+    <div class="container">
+        <h1>Countries of the World</h1>
+        <input type="text" id="country_Input"
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/countries.html]: IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: pepito_alonajane/countries.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="stylesheet" href="countries.css">
+    <title>Countries</title>
+</head>
+<body>
+    <div class="container">
+        <h1>Countries of the World</h1>
+        <input type="text" id="country_Input" 
+                placeholder="Please enter a country">
+        <button onclick="searchCountry()">Search</button>
+        <div class="country-details" id="country_Details"></div>
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/countries.html]: do not put id and class in one div

File: pepito_alonajane/countries.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <div class="same-region-countries" id="same_Region_Countries"></div>
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/countries.html]: -IDs should be written in snake case. All lower case separated by underscore.
-do not put id and class in one div
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p

File: pepito_alonajane/script.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    });
+    displayComments();
+  }
+});
+
+  function searchCountry() {
+    let countryName = document.getElementById
+    ('country_Input').value.trim();
+    if (!countryName) {
+      document.getElementById('country_Details').innerHTML =
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/script.js]: IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p

File: pepito_alonajane/script.js [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+  }
+});
+
+  function searchCountry() {
+    let countryName = document.getElementById
+    ('country_Input').value.trim();
+    if (!countryName) {
+      document.getElementById('country_Details').innerHTML = 
+          '<p>Please provide a country name.</p>';
+      document.getElementById('same_Region_Countries').innerHTML = '';
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/script.js]: IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p

File: pepito_alonajane/script.js [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+          `;
+      }).join('');
+      document.getElementById('same_Region_Countries').innerHTML = `
+          <h2>Countries in the Same Region (${region})</h2>
+          <div class="country-list">${sameRegionCountriesList}</div>
+      `;
+  })
+  .catch(function(error) {
+      console.error('Error fetching data:', error);
+      document.getElementById('country_Details').innerHTML
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/script.js]: IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p

File: pepito_alonajane/script.js [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+       = '<p>An error occurred: ' + error.message + '</p>';
+      document.getElementById('same_Region_Countries').innerHTML = '';
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/script.js]: IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p

File: pepito_alonajane/index.html [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+         <h2>My Goals</h2>
+            <ul>
+              <li>Finish my education.</li>
+              <li>Get a good job.</li>
+              <li>Open a Restaurant/Bakery</li>
+              <li>Give back to my parents.</li>
+              <li>Financial Stability.</li>
+            </ul>
+      </div>
+    </div>
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/index.html:45]: Observe proper indentation for your div tag

File: pepito_alonajane/index.html [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+     <div class="container">
+       <div class="section">
+         <h2>Comments</h2>
+            <button id="sort_asc_btn">
+            Sort by Date Ascending</button>
+            <button id="sort_desc_btn">
+               Sort by Date Descending</button>
+              <div id="commentsSection"></div>
```
Comment (Pasionpreciouskaye) [inline pepito_alonajane/index.html]: IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: pepito_alonajane/index.html [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+   <div class="contact">
+     <div class="header">
+        <h2> Contact Me</h2>
+     </div>
+         <form>
+            <div class="email">
+               <label for="email">Email:</label>
+               <input type="email" id="email" name="email">
+             </div>
+            <div class="full_name">
```
Comment (ElizaAlimasa) [inline pepito_alonajane/index.html]: Class names should be written in kebab case. All lower case separated by hyphen. For reference, please click on the link provided below to access further information.


```suggestion
            <div class="full-name">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

**Review by ElizaAlimasa — CHANGES_REQUESTED** [review]
Kindly review the suggested changes in your files and make sure to update your js and css file.

**Review by Pasionpreciouskaye — APPROVED** [review]
You can now merge your exercise 14

**Review by ElizaAlimasa — APPROVED** [review]
Everything looks good to me. You can now merge it to main.

**Pull request merged**